addons/mass_mailing/static/src/snippets/s_social_media_mass_mailing/options.js

```diff
@@ -0,0 +1,109 @@
+odoo.define('@web_editor/src/snippets/s_social_media_mass_mailing/options', function (require) {
+
+const options = require('web_editor.snippets.options');
+const {SocialMedia} = require('@web_editor/js/editor/snippets/s_social_media/options');
+const rpc = require('web.rpc');
+const session = require('web.session');
+
+let dbSocialValues;
+let dbSocialValuesProm;
+options.registry.SocialMediaMassMailing = SocialMedia.extend({
+    /**
+     * @override
+     */
+    async cleanForSave() {
+        await this._fetchSocialMedia();
```

Comment:
Why do you need this in `cleanForSave`? (is not it worth a comment in the code?)